cms/static/sass/views/_certificates.scss

```diff
@@ -125,15 +125,15 @@
 
       .actions {
         @include transition(opacity .15s .25s ease-in-out);
```

Comment:
Is this right? Shouldn't it be: `(property, duration, timing function, transition delay)`?